Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -242,172 +256,53 @@ edgebridgetutorialapp://
-| --------------------------------------- | ---------------------------------- | ------------------ | ------------------ |
-| data.contextdata.product\\.add\\.event  | commerce.productListAdds.value     | :white_check_mark: |                    |
-| data.contextdata.product\\.view\\.event | commerce.productListViews.value    |                    | :white_check_mark: |
-| data.contextdata.product\\.id           | productListItems.SKU               | :white_check_mark: | :white_check_mark: |
-| data.contextdata.product\\.name         | productListItems.name              | :white_check_mark: | :white_check_mark: |
-| data.contextdata.product\\.units        | productListItems.quantity          | :white_check_mark: |                    |
+#### Trigger rule-based `trackAction` events <!-- omit in toc -->
+Rule-based `trackAction`/`trackState` events are also converted to Edge events by the Edge Bridge extension. Select the **Trigger Consequence** button (**1**) to initiate a rule that generates a `trackAction` event.
 
-12. After completing all the mappings, click **Save**.
+<img src="../assets/edge-bridge-tutorial/assurance-validation/ios-app-trigger-rule-button.png" alt="Simulator tracking buttons" width="400"/>
 
-## Final validation using Assurance
-Now that the mapping is set up in the datastream, we have the full pathway of data:
+> [!TIP]
+> If no `trackAction` event is dispatched upon selecting the **Trigger Consequence** button, verify that your mobile property is [set up with the correct rule](#2-configure-a-rule-to-forward-personal-identifiable-information-pii-events-to-edge-network).
 
-```mermaid
-graph LR;
-    step1(App<br/>Analytics trackAction/trackState) --> step2(App<br/>Edge Bridge conversion to Edge event) --> step3(Edge Network<br/>Datastream translation of payload from contextdata to Edge XDM) --> step4(Edge Network<br/>Routing XDM data using datastream to Analytics);
-```
+Just like the `trackAction`/`trackState` events above, the Edge Bridge extension converts the personal PII `trackAction` event into an Edge event.
 
-By using the Event Transactions view in the left-side navigation panel, the logical flow of events from `trackAction` event -> data mapping -> Analytics can be seen.
-<img src="../assets/edge-bridge-tutorial/assurance-validation/assurance-event-transactions.png" alt="Assurance Event Transactions" width="1100"/>  
+With this, the migration from an existing Analytics implementation to sending data via Edge Network to Analytics, powered by Edge Bridge, is complete!
 
-Back in the Events view, the **mapper:xdm-event** (**1**) shows the result of the mapping from the Edge Bridge event's generic **data** -> **contextdata** structure into the XDM schema format (**2**).
-<img src="../assets/edge-bridge-tutorial/assurance-validation/assurance-mapper-event.png" alt="Assurance mapper event" width="1100"/>  
+## Optional - Data Prep for Data Collection mapping
```

Comment:
renaming suggestion: Optional - Map track event data with Data Prep for Data Collection